airbyte-integrations/connectors/source-postgres/src/testFixtures/java/io/airbyte/integrations/source/postgres/PostgresTestDatabase.java

```diff
@@ -19,8 +19,25 @@
 public class PostgresTestDatabase extends
     TestDatabase<PostgreSQLContainer<?>, PostgresTestDatabase, PostgresTestDatabase.PostgresConfigBuilder> {
 
-  static public PostgresTestDatabase in(String imageName, String... methods) {
-    final var container = new PostgresContainerFactory().shared(imageName, methods);
+  public static enum BaseImage {
+
+    POSTGRES_16_BULLSEYE("postgres:16-bullseye"),
```

Comment:
non-blocking request: could we have some javadoc commentary for each image with at least a hyperlink to its dockerhub page? Might be useful to help maintenance.

Reply:
there's really very little info in the individual image's dockerhub page. For example, [postgres:12-bullseye]([url](https://hub.docker.com/layers/library/postgres/12-bullseye/images/sha256-e0bc95703b924da949b2eb8b4ed5af8485b1f35b50c04577a0848b3e6a3a3abd?context=explore)) vs [postgres:16-bullseye]([url](https://hub.docker.com/layers/library/postgres/16-bullseye/images/sha256-e07f542bafb5c272aa2d2fc478b942fb33acb5f0db9e8f33bee15cc2340af951?context=explore)). In addition, the URL changes with each release of a new version of the image...